src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -267,6 +295,53 @@ public IActionResult OrganisationName(string post)
             return View("OrganisationName", organisationVm);
         }
 
+        [Route("TotalEmployees")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult TotalEmployees()
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
+
+            if (!_reportService.CanBeEdited(report))
+                return new RedirectResult(Url.Action("Index", "Home"));
+
+            var totalEmployeesVm = new TotalEmployees
+            {
+                HasTotalEmployeesMeetMinimum = report.HasTotalEmployeesMeetMinimum,
+                Report = report
+            };
+
+            ViewBag.CurrentPeriod = report?.Period ?? _currentPeriod;
+
+            return View("TotalEmployees", totalEmployeesVm);
+        }
+
+        [Route("PostTotalEmployees")]
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult PostTotalEmployees(TotalEmployees totalEmployeesVm)
+        {
+            var totalEmployeesViewModel = new TotalEmployees
```

Comment:
surely we should be validating that the user has pushed a response here. If I don't select either radio option I should be presented with a validation error. As-is is just handling it as if I said I have less than 250 employees